lib/handler.js

```diff
@@ -1,7 +1,7 @@
 const bunyan = require('alpha-lambda-bunyan');
 const DynamoDB = require('aws-sdk/clients/dynamodb');
 const lambdaHandler = require('alpha-lambda');
-const pick = require('lodash.pick');
+const pick = require('lodash/pick');
```

Comment:
How is this different from bringing the whole `lodash` library?

Reply:
It allows for tree-shaking to reduce the overall package size when bundling with webpack or a similar tool. Including the whole library like this:

```javascript
const { pick } = require('lodash');
```

causes the entire lodash library to be included in the bundle, whether it's actually used in application code or not.

Reply:
I get that, but it's still not ideal. I think there should be a defect raised against `lodash`.